SSDataSources.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.license      = { :type => 'MIT', :file => 'LICENSE'  }
   s.author       = { "Jonathan Hersh" => "jon@her.sh" }
   s.source       = { :git => "https://github.com/splinesoft/SSDataSources.git", :tag => s.version.to_s }
-  s.platform     = :ios, '6.0'
+  s.platform     = :ios, '8.0'
```

Comment:
Reverted to iOS 6.  No problem.
Mainly because less than 8% of users are running iOS < 9.0 (https://developer.apple.com/support/app-store/).